src/Native/pkg/runtime.native.System.IO.Compression/ubuntu/16.04/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <Version>1.0.1</Version>
+    <PackageTargetRuntime>ubuntu.16.04-$(PackagePlatform)</PackageTargetRuntime>
+    <!-- only build for x64 -->
+    <PackagePlatforms>x64;</PackagePlatforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <File Include="$(UbuntuNativePath)System.IO.Compression.Native.so">
```

Comment:
In the TFS build, this will cause us to package Ubuntu 14.04 resources in these packages (since the internal build defines this to the place the Ubuntu 14.04 native binaries are dropped there (see `Packaging.props` internally)

After we enable the VSO builds and start dropping stuff internally, we will need to change this so that we have a different property between Ubuntu 14.04 and Ubuntu 16.04.  Keeping it as it is now is good however, as it will keep the internal build happy.